gcloud/bigquery/job.py

```diff
@@ -927,6 +928,11 @@ def __init__(self, name, query, client):
     https://cloud.google.com/bigquery/docs/reference/v2/jobs#configuration.query.useQueryCache
     """
 
+    user_defined_function_resources = _TypedProperty('user_defined_function_resources', list)
+    """See:
+    https://cloud.google.com/bigquery/docs/reference/v2/jobs#configuration.query.userDefinedFunctionResources
+    """
```

Comment:
Using `_TypedProperty` for a list of mappings leaves us with no checking at assignment time.  @dhermes, do you think we should have a "setter" for the property which checks that the list elements are dicts with appropriate keys?